node.js

```diff
@@ -49,7 +49,42 @@ module.exports = {
     'space-before-function-paren': off,
     'keyword-spacing': error,
     'eol-last': [ error, always ],
-    'brace-style': [ error, '1tbs', { allowSingleLine: true } ],
-    'curly': [ error, all ]
+    'brace-style': [ error, '1tbs', { allowSingleLine: false } ],
+    'curly': [ error, all ],
+    'block-spacing': [ error,  always ],
+    'indent': [ error, 2, { SwitchCase: 1 } ],
+    'no-tabs': error,
+    'spaced-comment': [ error, always ],
+    'array-callback-return': warn,
+    'eqeqeq': [ error, always, { null: 'ignore' } ],
+    'no-else-return': [ warn, { allowElseIf: false } ],
+    'yoda': error,
+    'no-throw-literal': error,
+    'no-self-compare': error,
+    'no-return-assign': error,
+    'no-loop-func': error,
+    'valid-jsdoc': warn,
+    'for-direction': error,
+    'no-buffer-constructor': error,
+    'no-new-require': error,
+    'no-path-concat': error,
+    'comma-spacing': error,
+    'comma-style': error,
+    'computed-property-spacing': [ error, never ],
+    'consistent-this': [ error, 'self' ],
+    'func-call-spacing': [ error, never ],
+    'key-spacing': error,
+    'new-cap': error,
+    'no-lonely-if': error,
+    'no-mixed-operators': error,
+    'no-unneeded-ternary': [ error, { defaultAssignment: false } ],
+    'quote-props': [ warn, 'consistent-as-needed' ],
```

Comment:
Warn for inconsistent property quoting. Disallow quotes unless they're required by any properties, then require it for all properties in that object.
https://eslint.org/docs/rules/quote-props
```js
// Old
{
  foo: 'bar',  
  'baz': 42,
  'qux-lorem': true
};
var object2 = {
  'foo': 'bar',
  'baz': 42
};

// New
var object1 = {
  'foo': "bar",
  'baz': 42,
  'qux-lorem': true
};
var object2 = {
  foo: 'bar',
  baz: 42
};
```

Reply:
`foo` and `baz` do not need the quotes in `object1`